Point_set_processing_3/include/CGAL/Point_with_normal_3.h

```diff
@@ -147,7 +147,8 @@ struct Normal_of_point_with_normal_pmap
   typedef Normal_of_point_with_normal_pmap<Gt> Self;
   /// \name Put/get free functions
   /// @{
-  friend reference get(const Self&,const key_type& k) {return k.normal();}
+  friend const reference get(const Self&,const key_type& k) {return k.normal();}
```

Comment:
That code will break user-code that does:
```C++
  typedef typename boost::property_traits<Pmap>::reference reference;
  reference v = get(pmap, key);
```
That will break if `key` is `const`, because one cannot bind a const-ref to a ref.